docs/Authoring.md

```diff
@@ -155,17 +155,17 @@ dependency.
 
 For a given a target platform, NuGet will add references to all the libraries in
 the corresponding lib folder. You can override this behavior by using the
-`Reference` item:
+`LibraryReference` item:
```

Comment:
`Reference` has specific meaning in `Microsoft.Common.targtets`. I had to rename the item to avoid conflicts.